skyportal/handlers/api/tns.py

```diff
@@ -382,6 +382,8 @@ def post(self, obj_id):
             data = self.get_json()
             tnsrobotID = data.get('tnsrobotID')
             reporters = data.get('reporters', '')
+            archival = data.get('archival', False)
+            archivalComment = data.get('archivalComment', '')
```

Comment:
here I would do `archival_comment = data.get('archivalComment', '')` instead, to have that snake_case